test/Shared/MsBuildProjectSetupHelper.cs

```diff
@@ -58,10 +58,16 @@ public void SetupProjects(TemporaryFileProvider fileProvider, ITestOutputHelper
         {
             Directory.CreateDirectory(Path.Combine(fileProvider.Root, "Root"));
             Directory.CreateDirectory(Path.Combine(fileProvider.Root, "Library1"));
+            Directory.CreateDirectory(Path.Combine(fileProvider.Root, "toolAssets", "netcoreapp2.0"));
```

Comment:
Not netcoreapp2.1?

Reply:
well the assembly that gets put in there is a `netstandard` assembly. The directory structure is just to follow the one that the `TestCodeGeneration.targets` file expects.